hack/get-disk-skus.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	compute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-12-01/compute"
+	"io/ioutil"
+	"k8s.io/klog/v2"
+	"os"
+	"os/exec"
+	"strings"
+
+	azuredisk "sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk"
+)
+
+func init() {
+	klog.InitFlags(nil)
+}
+
+func main() {
+
+	skusFullfilePath := "skus-full.json"
```

Comment:
is it possible to define `diskSkus map[string]map[string]DiskSkuInfo` in go file directly, may reuse this map: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/pkg/azuredisk/azure_dd_max_disk_count.go, use `populateSkuMap` to parse from one file is error prone, if it's a built-in map, then would be quite straightforward.

Reply:
It's a well formed json, we are not parsing a flat file or random string. I don't think it's a good idea to hardcode an ever changing configuration. A new sku would mean we have to make a code change.
As mentioned in the other file, I plan to change this to take a dependency on the skus API.

I'll hardcode it for now, to simply deployments, since I plan to change this anyway.